bin/publish.sh

```diff
@@ -19,7 +19,10 @@ echo "Removing existing files"
 rm -rf public/*
 
 echo "Generating site"
-hugo
+hugo --minify
+
+echo "Recreating CNAME"
+cp CNAME public/
```

Comment:
It turned out to be a simpler fix than I expected - I ran publish a few times and copying the CNAME over seems to maintain the rotational.io setting. Also, there is a `cname` configuration in GitHub actions that I believe does the same thing. 

Reference: https://github.com/tschaub/gh-pages/issues/236